src/scene/graphics/quad-render-utils.js

```diff
@@ -68,8 +68,7 @@ function drawQuadWithShader(device, target, shader, rect, scissorRect) {
     // the same framebuffer. The workaround here is to store multi-sampled color buffer, instead of only resolving it, which is wasted
     // memory bandwidth. Without this we end up with a black result (or just UI), as multi-sampled color buffer is never written to.
     if (device.isWebGPU && target === null) {
-        const samples = target?.samples ?? device.samples;
-        if (samples > 1)
+        if (device.samples > 1)
```

Comment:
This is/was dead code, we are testing `target?.samples` even though the `if` makes sure that `target === null`

Reply:
nice catch, now might as well collapse the nested ifs?

Reply:
> nice catch, now might as well collapse the nested ifs?

nice observation, gonna do that